ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml

```diff
@@ -249,6 +249,18 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-usgov-ipi-private
+- as: e2e-azure-ipi-kubevirt-hypershift-mgmt-p2
+  cron: 0 4 */7 * *
+  steps:
+    cluster_profile: azure4
```

Comment:
@jianlinliu  Is `azure4` the expected one here ?

Reply:
We found only this profile could support kubevirt in prow. We will do more investigation here to apply qe's profile later.

Reply:
Is this urgent? Generally we are supposed to use QE's profile for testing.

Reply:
Yes, we'd like to add it to have a look at the status of e2e.  It's also a part of HyperShift kubevirt plan. After this PR, we will continue to do more on kubevirt including replacing QE's profile & aws platform support.

Reply:
Besides, it is easy to let kubevirt QE know the status of this pipeline and prepare their integration test based on this pipeline. Thanks!

Reply:
@jianlinliu could you help to review again? thanks^^